go-backend/db/organization.go

```diff
@@ -190,3 +212,13 @@ func (s *pgStore) GetOrganization(ctx context.Context, organizationID int) (orga
 
 	return
 }
+
+// GetOrganizationByDomainName - given a context and a string representing a domain name, look for the organization in the
+// database based on that domain name.
+func (s *pgStore) GetOrganizationByDomainName(ctx context.Context, domainName string) (organization Organization, err error) {
+	err = s.db.Get(&organization, getOrganizationByDomainNameQuery, domainName)
+	if err != nil {
+		logger.WithField("err", err.Error()).Error("Error selecting organization by domain name: " + domainName)
```

Comment:
missing `return`

Reply:
Nice catch, thank you. ba63ab38fd9d79d6b4fb1401be55a57540c4303a